### PR TITLE
Use pyproject.toml only (move away from setup.cfg) 

### PR DESCRIPTION
**Pull request by gsheni**

### CODE REVIEW

File: nlp_primitives/__init__.py [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+    if (
+        inspect.isclass(obj)
+        and obj is not AggregationPrimitive
+        and obj is not TransformPrimitive
+        and issubclass(obj, (AggregationPrimitive, TransformPrimitive))
+    )
+]
 
-nltk_data_path = pkg_resources.resource_filename("nlp_primitives", "data/nltk-data/")
+nltk_data_path = pkg_resources.resource_filename("nlp_primitives", "data/nltk_data/")
```
Comment (gsheni) [inline nlp_primitives/__init__.py:47]: Renamed this to underscore, since its more correct (Python wise)

File: nlp_primitives/required_data.txt [hunk 2]
```diff
@@ -1 +1 @@
-nltk-data.tar.gz
\ No newline at end of file
+nltk_data.tar.gz
```
Comment (gsheni) [inline nlp_primitives/required_data.txt:1]: @rwedge @thehomebrewnerd do we actually need this?

Comment (thehomebrewnerd) [reply nlp_primitives/required_data.txt]: Would have to study a bit more to know for sure, but it looks like we are including the data we need by including the `data/nltk_data` directory, so it doesn't seem like that file is needed for anything.

Comment (gsheni) [reply nlp_primitives/required_data.txt]: Removed

File: .flake8 [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[flake8]
+exclude = docs/*
+max-line-length = 88
+extend-ignore = E203
+ignore = E501,W504,W503
+per-file-ignores =
+    **/__init__.py:F401
+    **/tests/*:D
```
Comment (rwedge) [inline .flake8:8]: what is the D class of errors?

Comment (gsheni) [reply .flake8]: Removed:
https://github.com/alteryx/nlp_primitives/pull/201/commits/d5a5f3114685cdf36b79e258b4241c6c8b5486d2

**Review by rwedge — APPROVED** [review]

**Pull request merged**